.github/actions/store_artifacts/action.yml

```diff
@@ -0,0 +1,41 @@
+name: 'Store artifacts'
+description: 'Store given artifacts in a proper place on a shared drive'
+inputs:
+  artifacts:
+    description: "Multi-line list of artifacts to store"
+    required: true
+  storage_dir:
+    description: "Directory name to store artifacts in"
+    required: true
+  storage_root:
+    description: "Root path of the storage to place artifacts to"
+    required: true
+  action_path:
+    description: "Action path, if not set - taken from github context"
+    required: false
+
+
+outputs:
+  artifacts_storage_path:
+    description: "Path where the artifacts are stored"
+    value: ${{ steps.copy_artifacts.outputs.artifacts_storage_path }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install Python dependencies
+      uses: py-actions/py-dependency-install@v4
+      with:
+        path: "${{ inputs.action_path && inputs.action_path || github.action_path }}/requirements.txt"
+        update-setuptools: "false"
+        update-wheel: "false"
+
+    - name: 'Copy artifacts'
+      id: copy_artifacts
+      shell: ${{ runner.os == 'Windows' && 'pwsh' || 'bash' }}
+      run: >-
+        python ${{ inputs.action_path && inputs.action_path || github.action_path }}/store_artifacts.py
```

Comment:
the same question about action_path, why do we need it?

Reply:
Due to issues with workspace paths on Azure runners